builtin/credential/aws/path_config_identity.go

```diff
@@ -18,6 +18,11 @@ func pathConfigIdentity(b *backend) *framework.Path {
 				Default:     identityAliasIAMUniqueID,
 				Description: fmt.Sprintf("Configure how the AWS auth method generates entity aliases when using IAM auth. Valid values are %q and %q", identityAliasIAMUniqueID, identityAliasIAMFullArn),
 			},
+			"ec2_alias": {
```

Comment:
Since the above parameter has the old style of declaring the parameter type (`&framework.FieldSchema`), could we run `gofmt -s` on this file since we're here anyway? That would clean up the above and any other places that need updating in the file.